tools/cli-core/src/git.ts

```diff
@@ -132,7 +132,10 @@ export const sparseCheckoutRepo = async (
  * @return {Response<string>} - the simple-git response.
  */
 export const checkoutRef = ( pathToRepo: string, ref: string ) => {
-	const git = simpleGit( { baseDir: pathToRepo } );
+	const git = simpleGit( {
+		baseDir: pathToRepo,
+		config: [ 'core.hooksPath=/dev/null' ],
```

Comment:
I've chosen this solution to disable hooks in the code analyzer because it doesn't assume presence of husky and in some cases we don't install the husky hooks when doing a sparse checkout of a repo.